ultraplot/axes/geo.py

```diff
@@ -65,6 +71,18 @@
 longridminor, latgridminor, gridminor : bool, default: :rc:`gridminor`
     Whether to draw "minor" longitude and latitude lines.
     Use the keyword `gridminor` to toggle both at once.
+tick : bool or float, optional
+    Whether to draw axis tick marks and their length in points. If `True`, uses the default
+    tick length from rcParams. If `False` or `None`, ticks are not drawn. If a numeric value
+    is provided, ticks are drawn with that length.
+lontick : bool or float, optional
+    Controls tick marks specifically for longitude. Takes precedence over `tick` parameter.
+    If `True`, uses the default tick length. If `False` or `None`, longitude ticks are not drawn.
+    If a numeric value is provided, longitude ticks are drawn with that length in points.
+lattick : bool or float, optional
+    Controls tick marks specifically for latitude. Takes precedence over `tick` parameter.
+    If `True`, uses the default tick length. If `False` or `None`, latitude ticks are not drawn.
+    If a numeric value is provided, latitude ticks are drawn with that length in points.
```

Comment:
The keyword argument names and semantics for these tick marks needs to follow the same API as done in format for `x` and `y` tick marks. See the doc string here: https://github.com/Ultraplot/ultraplot/blob/main/ultraplot/axes/cartesian.py#L38.